### PR TITLE
LanguageTool: discontinue packaging like Arch

### LINKED ISSUE #42372: LanguageTool does not start with TeXstudio
### Is this a new report?

Yes

### System Info

Void 6.1.12_1 x86_64 GenuineIntel notuptodate rFF

### Package(s) Affected

LanguageTool-5.8_1, texstudio-4.4.1_1

### Does a report exist for this bug with the project's home (upstream) and/or another distro?

_No response_

### Expected behaviour

Opening TeXstudio, configure "LT Path" to `/usr/share/java/languagetool/languagetool-server.jar`.  Clicking Help -> Check LanguageTool is expected to show that LanguageTool is configured correctly:
```
which java: /usr/bin/java

JAVA: java
openjdk version "11.0.12" 2021-07-20
OpenJDK Runtime Environment (build 11.0.12+7-void-r3)
OpenJDK 64-Bit Server VM (build 11.0.12+7-void-r3, mixed mode)

Real-time checking is enabled.
Grammar checking is enabled.

Tries to start automatically.

LT current status: working

LT-URL: http://localhost:8081/v2/check
```

### Actual behaviour

Instead:
```
which java: /usr/bin/java

JAVA: java
openjdk version "11.0.12" 2021-07-20
OpenJDK Runtime Environment (build 11.0.12+7-void-r3)
OpenJDK 64-Bit Server VM (build 11.0.12+7-void-r3, mixed mode)

Real-time checking is enabled.
Grammar checking is enabled.

Tries to start automatically.

LT current status: error


LT-URL: http://localhost:8081/v2/check
```
Executing the jar command to start LanguageTool shows:
```
$ languagetool --http -p 8081                                             INT
2023-02-21 13:52:51 +0000 Setting up thread pool with 10 threads
WARN: no common words file defined for Japanese - this language might not be correctly auto-detected
WARN: no common words file defined for Khmer - this language might not be correctly auto-detected
2023-02-21 13:52:54 +0000 Starting LanguageTool 5.8 (build date: 2022-07-02 14:50:13 +0000, 82e790a) server on http://localhost:8081...
2023-02-21 13:52:54 +0000 Server started
^C%                                                                             ~ > java -jar /usr/share/java/languagetool/languagetool-server.jar      INT 38s
Exception in thread "main" java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory
	at org.languagetool.server.Server.<clinit>(Server.java:40)
Caused by: java.lang.ClassNotFoundException: org.slf4j.LoggerFactory
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	... 1 more
```
On the other hand, running the Void packaged LanguageTool binary from the command line (`languagetool --http -p 8081`) does work.

### Steps to reproduce

1. Start TeXstudio
2. Add path to languagetool-server.jar to configuration.
3. Check LanguageTool from TeXstudio

### PR DESCRIPTION
**Pull request by Ophidiophobia**
Template of LanguageTool followed Arch's [Java Package Guidelines](https://wiki.archlinux.org/title/Java_Package_Guidelines) and put files in several places. Arch's [bin wrapper](https://github.com/archlinux/svntogit-community/blob/master/languagetool/trunk/languagetool.sh) "fixed" all issues by defining (many) class paths. This approach fails if other tools try to start the jars directly without doing those "fixes" (see #42372).

Use four very simple binary script wrappers tracked in our git instead.

Install LanguageTool in one single directory.
This change makes LanguageTool behave like upstream and easier to configure and use in other applications like TeXstudio. Fixes #42372 "LanguageTool does not start with TeXstudio"

####Possible points of interests
- /usr/bin/languagetool starts the gui variant because that seems to be the version when upstream says to start the standalone version.
- languagetool-cli : -cmd or -command-line instead of -cli?
- languagetool-http(s): -server is ambigious and -server-http(s) too long
- commit message is too wordy but I am not sure which parts to throw away

#### Testing the changes
- I tested the changes in this PR: **briefly** (issue use case, "--help", checked and corrected commit message using languagetool-cli)

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-libc)

### CODE REVIEW

**Comment by Eloitor** [comment]
Maybe we could build LanguageTool from source in this PR or is it preferable to wait for this PR to be merged?

https://github.com/void-linux/void-packages/issues/40910

**Comment by Ophidiophobia** [comment]
Honestly I am in favor of switching to actually building from source and also upgrade to 6.0. I think the template should never had pulled in the dist files but here we are.
The main reason I did not go that route are:
- not sure how long compiling would take
- unsure which version LanguageTool actually ships on their website
- not sure if I would introduce any weird behaviour by unwittingly crate something slightly different
- we can have a long debate if LanguageTool (or any java application) should use dependencies from common sources (we have java-commons-io) and run into java's version of dll hell and maybe reintroduce class path hell or keep it sane and simple and have each app keep dependencies in their own directory

So yeah. Compiling from source is better in my opinion but I wanted to fix packaging quickly so users can easily use LanguageTools from other apps again.

**Comment by Ophidiophobia** [comment]
sigh I need to express myself better:
Merge this first (if there are no further objections). It makes it a lot easier for unaware users to use LanguageTools in other apps without workarounds.

How to build and package this from source can be debated later (in a second PR).
